go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/keboola/keboola-as-code
 
 go 1.17
 
+replace github.com/google/go-jsonnet v0.18.0 => github.com/keboola/go-jsonnet v0.18.1-0.20220304084238-c20b815c4986
+
```

Comment:
V `go.mod` je mozne urobit replace.

Reply:
🤔 takže díky tomu v kódu zůstává původní reference na `google/go-jsonnet` i když se namísto ní používá `keboola/go-jsonnet`? Je to potřeba? Aby to časem nebylo zdrojem nějakých zmatků. Když do toho přijde někdo novej tak aby se pak nedivil třeba.

Reply:
No ak to tak nebude, tak budem musel prerobit cely `fork`, aby tam nebolo `/google` ale `/keboola` v kazdom importe, v repe https://github.com/keboola/go-jsonnet

Tj. takto vo forku nemusim menit nazov balicka tu: https://github.com/keboola/go-jsonnet/blob/c20b815c4986b574d070ef22e47d298010fefb07/go.mod#L1

A viem to hocikedy rebasnut nad original (je tam teda ovela menej zmien, ziadne zmeny v importoch).

Myslim, ze toto je to spravne riesenie pre fork. V IDE ta to presmeruje na spravny kod.

Reply:
Je to aj v oficialnej dokumentacii `Replacing with a fork of the module repository`:
https://go.dev/doc/modules/gomod-ref

Reply:
A jo, tak to je jasný, to nechceme. 😃